.github/policies/resourceManagement.yml

```
@@ -77,18 +77,10 @@ configuration:
       - labelAdded:
           label: breaking-change
       then:
-      - addLabel:
-          label: needs-breaking-change-doc-created
       - addReply:
           reply: >-
-            Added `needs-breaking-change-doc-created` label because this issue has the `breaking-change` label. 
-
-
-            1. [ ] Create and link to this issue a matching issue in the dotnet/docs repo using the [breaking change documentation template](https://aka.ms/dotnet/docs/new-breaking-change-issue), then remove this `needs-breaking-change-doc-created` label.
-
-
-            Tagging @dotnet/compat for awareness of the breaking change.
-      description: Add breaking change doc label to issue
+            Refer to the [.NET SDK breaking change guidelines](https://github.com/dotnet/sdk/blob/main/documentation/project-docs/breaking-change-guidelines.md)
```

```suggestion
            Refer to the [.NET SDK breaking change guidelines](https://github.com/dotnet/sdk/blob/main/documentation/project-docs/breaking-change-guidelines.md#required-process-for-all-net-sdk-breaking-changes)
```

This section specifically is the action-list for anyone making breaking changes - instead of getting lost in the doc it might be better to point contributors to the action list directly, and then they can use the rest of the doc as a reference.